README.md

```diff
@@ -131,6 +132,30 @@ export class Shout extends Component<ShoutSignature> {
 
 [yieldable named blocks rfc]: https://github.com/emberjs/rfcs/blob/master/text/0460-yieldable-named-blocks.md
 
+#### Template Components
+
+In GlimmerX, standalone templates can be invoked as components. Like class-based components, you can declare a signature for a template component in order for Glint to understand how it can be used.
+
+The class-based component above could as a template component like so:
+
+```ts
+import { TemplateComponent } from '@glint/environment-glimmerx/component';
```

Comment:
Suggestion: we should recommend `import type` over `import` wherever we're importing a type-only type, just for clarity.

```suggestion
import type { TemplateComponent } from '@glint/environment-glimmerx/component';
```